components/DesktopSidebar.tsx

```diff
@@ -9,26 +9,16 @@ import {
   UsersIcon,
 } from "@heroicons/react/24/outline";
 
-const navigation = [
-  { name: "Home", href: "/", icon: HomeIcon, current: true },
-  { name: "Login", href: "/api/auth/login", icon: UsersIcon, current: false },
-  {
-    name: "Logout",
-    href: "/api/auth/logout",
-    icon: FolderIcon,
-    current: false,
-  },
-  { name: "Profile", href: "/profile", icon: CalendarIcon, current: false },
-];
+const navigation = [{ name: "Home", href: "/", icon: HomeIcon, current: true }];
 
 const userNavigation = [
   { name: "Your profile", href: "/profile" },
   { name: "Logout", href: "/api/auth/logout" },
 ];
```

Comment:
What do you think about moving userNavigation and navigation into the `util` folder?